### ISSUE TITLE
Various NPCs: Outzone / SL and Symp Gain

### ISSUE THREAD

**Issue by Corruptforce** [NCC-Lykos/neocron-classic#231]
Quite a few NPCs in which appear to have neutral standing (yet hostile faction) are giving large amounts of SL and faction symp.

For example, TG contacts in PP3
Jade in OZ (can't remember which sector)

There isn't consistency with this, I suspect due to hostile faction they are giving rewards.
However other hidden faction neutral NPCs in the hidden passageway (which also appeared as negative SL) are giving symp loss and SL loss (which is what I would expect for quest, non hostile NPCs).

This is with Tangent, I will test results with other factions.

**Comment by NCC-Lykos** [comment]
So while this isn't a bug, we will have to make some adjustments to the way NPCs and SL and FS work.

**Comment by Corruptforce** [comment]
Sounds like a plan, heavily exploitable in the current state.

**Comment by Pantho** [comment]
Remove symp lose from these too...

My PPU got to -15 when he got a bit frisky with a beggar.

**Comment by Corruptforce** [comment]
I agree and disagree - there should be some deterrant from obliterating NPCs which don't tend to defend themselves/are static, however they shouldn't be as harsh potentially?

**Comment by M-ace** [comment]
i dont believe u should get neg or possitive sl fac symp from any npc that doesnt give money or xp.  those lil beggars and ho's are there for decoration and ambiance and get killed all the time accidentally.  if u are fighting around them and someone goes into stealth, the counter for stealth is aoe.  one aoe shot will kill those npc's and the aoe'r will of course loose/gain sL.  so i say they should be disabled.

**Comment by Pantho** [comment]
Yes, agree.

They're so low health they die to easily and have no sign as to there
faction affiliation.

**Comment by NCC-Lykos** [comment]
The following changes will be happening with this issue

1. NPCs will no longer effect soul light positive or negative
2. NPCs will no longer reward faction sympathy, but you can still lose faction sympathy for killing them

**Comment by Pantho** [comment]
Losing Sympathy would make sense - if only we knew what faction they belong
to?
They don't have signs showing you what faction they belong to.

**Comment by NCC-Lykos** [comment]
That can be corrected of course for NPCs that it matters on. Losing FS with Trader Union won't matter much :D
